src/datamodel/character/test/test_assassin.rs

```diff
@@ -0,0 +1,73 @@
+use crate::datamodel::character::*;
+
+//test ASSASSIN
+//test constructor
```

Comment:
I do think these kind of comments are redundant now that the file name and test method name indicate all useful information. I would remove all of them.

Reply:
I agree.